h2/src/main/org/h2/fulltext/FullText.java

```diff
@@ -751,30 +751,41 @@ protected static void addWords(FullTextSettings setting,
      * @param schema the schema name
      * @param table the table name
      */
-    protected static void createTrigger(Connection conn, String schema,
-            String table) throws SQLException {
+    private static void createTrigger(Connection conn, String schema,
+                                      String table) throws SQLException {
         createOrDropTrigger(conn, schema, table, true);
     }
 
     private static void createOrDropTrigger(Connection conn,
             String schema, String table, boolean create) throws SQLException {
-        Statement stat = conn.createStatement();
-        String trigger = StringUtils.quoteIdentifier(schema) + "."
-                + StringUtils.quoteIdentifier(TRIGGER_PREFIX + table);
-        stat.execute("DROP TRIGGER IF EXISTS " + trigger);
-        if (create) {
-            StringBuilder buff = new StringBuilder("CREATE TRIGGER IF NOT EXISTS ");
-            // needs to be called on rollback as well, because we use the init
-            // connection do to changes in the index (not the user connection)
-            buff.append(trigger).
-                append(" AFTER INSERT, UPDATE, DELETE, ROLLBACK ON ").
-                append(StringUtils.quoteIdentifier(schema)).
-                append('.').
-                append(StringUtils.quoteIdentifier(table)).
-                append(" FOR EACH ROW CALL \"").
-                append(FullText.FullTextTrigger.class.getName()).
-                append('\"');
-            stat.execute(buff.toString());
+        try (Statement stat = conn.createStatement()) {
+            String trigger = StringUtils.quoteIdentifier(schema) + "."
+                    + StringUtils.quoteIdentifier(TRIGGER_PREFIX + table);
+            stat.execute("DROP TRIGGER IF EXISTS " + trigger);
+            if (create) {
+                ResultSet rs = stat.executeQuery("SELECT value" +
+                        " FROM information_schema.settings" +
+                        " WHERE name = 'MV_STORE'");
```

Comment:
this query doesn't seem necessary now that you call FullTextTrigger.isMultiThread on the next line?